CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.7.0
+- Allow attaching multiple tags on failure. Changed type of `tag_on_failure` from string to array [#92](https://github.com/logstash-plugins/logstash-filter-kv/issues/92)
```

Comment:
A changelog indicating that we are changing types can be scary, and can lead people to believe that this is a breaking change when we are intending to implement this in a backward-compatible way. Let's rephrase it to indicate that this change allows a user to specify multiple tags.

```suggestion
- Allow attaching multiple tags on failure. The `tag_on_failure` option now also supports an array of strings [#92](https://github.com/logstash-plugins/logstash-filter-kv/issues/92)
```